docs_app/src/app/custom-elements/migration-timeline/Readme.md

```diff
@@ -0,0 +1,75 @@
+
+- 6.0.0-tenacious-rc.2_deprecation-class-AjaxObservable
```

Comment:
What's that for`?

Reply:
Deleted here:
dea4485

related to remove the comments